### DISCUSSION TITLE
Since release 2022.2.14 the "show source" button does not appear

### DISCUSSION THREAD

**Discussion by pvergain** [pradyunsg/furo#443] [Potential Issue] [answered]
Hello and thank you for the furo theme !

Since the 2022.2.14 release the "show source" button does not appear.

Reproduction
============

- clone the https://github.com/Mariatta/sphinx-intro-tutorial.git
- Install the last furo release (2022.4.7)
-  produce the documentation

We can see that there is no "Show button"
![image](https://user-images.githubusercontent.com/57228/169024755-b38ac851-3824-450a-b19b-45632f6cacc6.png)

- install the 2022.1.2 furo version
-  produce the documentation

And now we have the "Show source button"

![image](https://user-images.githubusercontent.com/57228/169025024-658540c5-cd7e-4d51-bdba-c614d6b486ec.png)

So I use the 2022.1.2 release.

**Comment by pradyunsg** [comment, marked as answer]
See https://github.com/pradyunsg/furo/discussions/374
